src/test/java/com/iexec/standalone/worker/WorkerServiceTests.java

```diff
@@ -59,9 +61,12 @@ class WorkerServiceTests {
 
     private static final String WORKER1 = "0x1a69b2eb604db8eba185df03ea4f5288dcbbd248";
     private static final String WORKER2 = "0x2ab2674aa374fe6415d11f0a8fcbd8027fc1e6a9";
+    private static final String WORKER3 = "0x3bo0902tn374fe7935d11f0a8fcbd1385fc1e6a6";
 
     @Container
     private static final MongoDBContainer mongoDBContainer = new MongoDBContainer(DockerImageName.parse(System.getProperty("mongo.image")));
+    @Mock
+    private TeeSconeService teeSconeService;
```

Comment:
I would prefer to see this declared with other mocks and to avoid hiding it in the middle of testcontainers related configurations